server/issue_test.go

```diff
@@ -279,6 +280,255 @@ func TestRouteShareIssuePublicly(t *testing.T) {
 	}
 }
 
+func TestShouldReceiveNotification(t *testing.T) {
+	s := ConnectionSettings{}
+	falseValue := false
+	tests := map[string]struct {
+		role         string
+		notification bool
+	}{
+		subCommandAssignee: {
+			role:         subCommandAssignee,
+			notification: falseValue,
+		},
+		subCommandMention: {
+			role:         subCommandMention,
+			notification: falseValue,
+		},
+		subCommandReporter: {
+			role:         subCommandReporter,
+			notification: falseValue,
+		},
+		subCommandWatching: {
+			role:         subCommandWatching,
+			notification: falseValue,
+		},
+	}
+	for name, tt := range tests {
+		t.Run(name, func(t *testing.T) {
+			val := s.ShouldReceiveNotification(
+				tt.role,
+			)
+			assert.Equal(t, tt.notification, val)
+		})
+	}
+
+}
+
+func TestFetchConnectedUser(t *testing.T) {
+	tests := map[string]struct {
+		p           *Plugin
+		instanceID  types.ID
+		client      Client
+		connection  *Connection
+		wh          webhook
+		expectedErr error
+	}{
+		"No Field": {
+			p: &Plugin{
+				instanceStore: mockInstanceStore{},
+				userStore:     mockUserStore{},
+			},
+			instanceID: testInstance1.InstanceID,
+			client:     nil,
```

Comment:
Here create more sub test cases like when client is nil and not nil similarly connection is nil and not nil

Reply:
It  is already there, by the way added 1 more test case in this function.